cmd/gitbase/server.go

```diff
@@ -1,4 +1,4 @@
-package cmd
+package main
```

Comment:
Why? Subcommnads are supposed to be implemented in a subpackage.

Reply:
Why do they need to be in a subpackage?

Reply:
Testing.